db/migrate/20201111122823_backfill_updated_at_of_ended_subscriptions.rb

```diff
@@ -0,0 +1,11 @@
+class BackfillUpdatedAtOfEndedSubscriptions < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
```

Comment:
Minor: you could mention why this is here in the commit - I'm assuming it's because the update is idempotent.

Reply:
I've updated the commit message. My thinking was as there is 629,854 records to update, if there is a transient failure along the way we won't have to start over and could just re-run the migration where it left off.